doc/ref_model/chapters/chapter04.md

```diff
@@ -268,60 +268,162 @@ Table 4-10 shows performance measurement capabilities.
-|---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Compute Flavour Name                  | A descriptive name                                                                                                                                                                                      |
-| Number of vCPUs | Number of virtual compute resources (vCPUs) presented to the VM instance.                                                                                                                               |
-| Memory                                | Virtual compute instance memory in megabytes.                                                                                                                                                           |
-| Ephemeral/Local Disk                  | Specifies the size of an ephemeral data disk that exists only for the life of the instance. Default value is 0.<br />The ephemeral disk may be partitioned into boot (base image) and swap space disks. |
-| Management Interface                  | Specifies the bandwidth of management interface/s                                                                                                                                                       |
+<p align="center"><img src="../figures/RM-ch-04-Workloads-Profiles-Flavours.png" alt="workload_design" title="Workload Design" width="65%"/></p>
+<p align="center"><b>Figure 4-3:</b>Workloads built against Cloud Infrastructure Profiles and Workload Flavours.</p>
 
-<p align="center"><b>Table 4-12:</b> Compute Flavour Geometry Specification.</p>
+A node configuration can be specified using the syntax:
 
-<a name="4.2.1.1"></a>
-#### 4.2.1.1 Predefined Compute Flavours
-The intent of the following Flavours list is to be comprehensive and yet effective to cover both IT and NFV workloads. The compute Flavours are specified relative to the “large” Flavour. The “large” Flavour configuration consists of 4 vCPUs, 8 GB of RAM and 80 GB of local disk, and the resulting virtual compute instance will have a management interface of 1 Gbps. The “medium” Flavour is half the size of a large and small is half the size of medium. The tiny Flavour is a special sized Flavour.
+>  \<profile name>[.\<profile_extension>][.\<extra profile specs>]
+
+where the specifications enclosed within "[" and "]" are optional, and the 'extra profile specs' are needed to capture special node configurations not accounted for by the profile and profile extensions.
+
+Examples, node configurations specified as: B, B.low-latency, H,  and H.very-high-speed-network.very-low-latency-edge.
```

Comment:
@pgoyal01 
As mentioned just before, Basic profile is for workload that can tolerate variable latency. The example B.low-latency is not consistent.

Reply:
@karinesevilla I tried to conform to the content in [RM 2.4.2](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#242-node-flavours-specialisations) as they were already approved and merged.

Could we have the situation where an application can request "low latency" to be placed in a site closer to the user but where this latency is not guaranteed.  The application would be designed to handle "longer" latency times.